Logging/ARKLogMessage.h

```diff
@@ -27,15 +27,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface ARKLogMessage : NSObject <NSCopying, NSSecureCoding>
 
+/// @warning Do not use this initializer.
+/// @raises NSInternalInconsistencyException This is not a valid initializer.
+- (instancetype)init;
```

Comment:
Before you were already hanging yourself by calling init on a log message, silently creating a useless, empty object. Now this makes your likely programming error more obvious.